conjure-java-core/src/main/java/com/palantir/conjure/java/services/dialogue/AsyncGenerator.java

```diff
@@ -152,6 +152,12 @@ public MethodSpec generate(ServiceDefinition def) {
                 .build());
     }
 
+    // TODO(ckozak): Type should be dealiased
```

Comment:
we basically have an alias resolving visitor in this class already so might as well implement

Reply:
:+1: updated